### ISSUE TITLE
Overriding a primitive converter (UUID)

### ISSUE THREAD

**Issue by mplokita-figure** [papsign/Ktor-OpenAPI-Generator#119]
I'm hoping to override the default behavior of zeroing an invalid UUID string. (For example, I might prefer to throw an exception or return null.)

Does this constitute a new feature request?

```kotlin
primitive {
    try {
        UUID.fromString(it)
    } catch (e: IllegalArgumentException) {
        null
    } ?: UUID(0, 0)
}
```

**Comment by Wicpar** [comment]
you can use a nullable uuid parameter, it uses 0 as a default if you don't allow it to be null.

**Comment by mplokita-figure** [comment]
@Wicpar Won't that affect the API spec? If we would like to communicate to readers of the spec that the UUID _is not optional_, then is there an alternative solution?

**Comment by Wicpar** [comment]
You are right, there is no way to throw an error doing that, the correct implementation would be to remove the try-catch and have an exception handler catch and build a proper error response

**Comment by aphu-figure** [comment]
@Wicpar As far as overriding default converter behavior, I'm assuming this is still a viable solution? Any plans to incorporate this?

https://github.com/papsign/Ktor-OpenAPI-Generator/issues/82#issuecomment-733191825

**Comment by Wicpar** [comment]
I'll accept the PR if you make one :)
